src/SFA.DAS.Forecasting.Api/Startup.cs

```diff
@@ -0,0 +1,124 @@
+            services.Configure<AzureActiveDirectoryConfiguration>(Configuration.GetSection("AzureAd"));
+            services.AddSingleton(cfg => cfg.GetService<IOptions<AzureActiveDirectoryConfiguration>>().Value);
+
+            var serviceProvider = services.BuildServiceProvider();
+            var azureActiveDirectoryConfiguration = serviceProvider.GetService<IOptions<AzureActiveDirectoryConfiguration>>();
+            var forecastingConfiguration = serviceProvider.GetService<IOptions<ForecastingConfiguration>>();
+
+            if (!Configuration["Environment"].Equals("LOCAL", StringComparison.CurrentCultureIgnoreCase))
+            {
+                services.AddAuthorization(o =>
+                {
+                    o.AddPolicy("default", policy =>
+                    {
+                        policy.RequireAuthenticatedUser();
+                    });
+                });
+                services.AddAuthentication(auth =>
+                {
+                    auth.DefaultScheme = JwtBearerDefaults.AuthenticationScheme;
+
+                }).AddJwtBearer(auth =>
+                {
+                    auth.Authority = $"https://login.microsoftonline.com/{azureActiveDirectoryConfiguration.Value.Tenant}";
+                    auth.TokenValidationParameters = new Microsoft.IdentityModel.Tokens.TokenValidationParameters
+                    {
+                        ValidAudiences = new List<string>
+                        {
+                            azureActiveDirectoryConfiguration.Value.Identifier,
+                            azureActiveDirectoryConfiguration.Value.Id
+                        }
+                    };
+                });
+                services.AddSingleton<IClaimsTransformation, AzureAdScopeClaimTransformation>();
+            }
+
+            services.AddMediatR(typeof(GetAccountExpiringFundsQueryHandler).Assembly);
+            services.AddScoped(typeof(IValidator<GetAccountExpiringFundsQuery>), typeof(GetAccountExpiryValidator));
+            services.AddTransient<IAccountProjectionRepository, AccountProjectionRepository>();
+            services.AddTransient<IAccountProjectionService, AccountProjectionService>();
+
+            services.AddMvc(o =>
+            {
+                if (!Configuration["Environment"].Equals("LOCAL", StringComparison.CurrentCultureIgnoreCase))
+                {
+                    o.Filters.Add(new AuthorizeFilter("default"));
+                }
+
+
+            }).SetCompatibilityVersion(CompatibilityVersion.Version_2_2);
+            
+            services.AddDbContext<ForecastingDataContext>(options => options.UseSqlServer(forecastingConfiguration.Value.ConnectionString));
+            services.AddScoped<IForecastingDataContext, ForecastingDataContext>(provider => provider.GetService<ForecastingDataContext>());
+        }
+
+        // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
```

Comment:
We can probably remove all the boilerplate comments as they are probably not needed